### DISCUSSION TITLE
Concern about multi account

### DISCUSSION THREAD

**Discussion by newjks** [sunflower-land/sunflower-land#2063] [General]
Hi, i'm a player just started to play the game last year november, regarding the multi account, recently my game pop up a screen mention about the game rule: multi account is banable, this screen was didn't show up when i started to play this game back to last year Nov. 

Back to this early month i created a new account with different wallet thinking to transfer resrouce to my main account, and thinking this is a smart strategy ( i haven't transfer any resource between them ), untill recently the pop up game rule and going to deep looking i just know this is not allowed and bannable, now even i haven't get ban but i'm seriously worry about this. i've invested my 400usd in NFT and worry all my NFT get ban hence didn't cont to invest more time in this game.

What should i do now, i love this game and i suggest put the game rule everytime we create a new game or starting to play, or just don't allow a player to create multiple account by detecting same PC or just disable ppl transfering resource.

**Comment by Ogrekin** [comment]
https://docs.sunflower-land.com/support/terms-of-service    Did you read the terms when you started playing multi account is strictly forbidden and a violation on the terms and term should you ALWAYS read...
I'm not one of the team but you have a big problem if you get caught so i recommend you burn 1 farm direct and i hope you didn't send sfl, resources or other stuff between the accounts and hope fore the best...
But clearly you have done wrong and i hope  you get it sorted out but sunflower-land policy is that there is no excuses about not seen info about since it in terms, "login to another farm is multi" and ban

**Reply by Ogrekin** [reply]
![image](https://user-images.githubusercontent.com/120697821/213694109-c7f00a11-8d88-4f41-8327-97c49235c89a.png)

**Reply by Ogrekin** [reply]
withdraw and use niftyswap to sell resources is available and should be and with that come send crops to other farms...
There  is no excuse for not reading the terms....  "i didn't know" doesn't work to say...

**Reply by newjks** [reply]
Thanks for the suggestion, how do i burn the other 1 account?

Yeah i took a good look into it few days ago after the game rule popup, and i didn't send slf or any resource between land ( was planning and lucky i read the rule ), i know niftyswap as i always use, i just thought i can outsmart other by creating another account, 

And i'm fine if they ban me now rather than later, since i haven't invest too much on it, and like just played 2 months ago, i'm just worried if i continue too play and invest other NFTs and get ban in the future. And i also hope they put the rule before starting a new game.

**Reply by Ogrekin** [reply]
If you want to burn you send it to an dead address like 0x00000_dead and the right one can you find in docs i think it was down by contributing at the bottom of the left column.

**Reply by newjks** [reply]
> If you want to burn you send it to an dead address like 0x00000_dead and the right one can you find in docs i think it was down by contributing at the bottom of the left column.

ok thanks!
